scripts/apps/dashboard/controllers/DashboardController.ts

```diff
@@ -22,7 +22,13 @@ export function DashboardController($scope, desks, dashboardWidgets, api, sessio
             $scope.$applyAsync(() => {
                 self.current = workspace;
                 self.widgets = extendWidgets(workspace.widgets || []);
-                self.availableWidgets = dashboardWidgets;
+                self.availableWidgets = dashboardWidgets
+                    .map((widget) => ({
+                        ...widget,
+                        descriptionHtml: widget.descriptionHtml
```

Comment:
can you avoid including `descriptionHtml` key to the mapped result if there's no value?